purepyvmware/datastores.py

```diff
@@ -0,0 +1,329 @@
+import time
+
+from lib import datastore_utils
+from pyVmomi import vim
+from pyVmomi import sms
+
+
+class Datastores(object):
+    """Maintenance of Pure Storage based datastores and associated array volumes.
+
+    For general maintenance of VMware datastores that are backed by Pure Storage FlashArrays.
+    """
+
+    def __init__(self, vsphere_content, flasharray):
+        """Setting up vSphere and FlashArray environment for use.
+
+        Args:
+            vsphere_content (vim.ServiceInstanceContent): All available content within the vSphere env.
+            flasharray (purestorage.FlashArray): Existing session to FlashArray.
+        """
+        self.flasharray = flasharray
+        self.pure_identifier = '24a937'
+        self.vsphere_content = vsphere_content
+
+    def create_compute_container_view(self):
+        """Create compute container view for an inventory of managed object references.
+
+        Returns:
+            resources (vim.view.ContainerView): Object view of compute resources.
+        """
+        compute_container = self.vsphere_content.viewManager.CreateContainerView(self.vsphere_content.rootFolder,
```

Comment:
Nit: as per above comment, break this out into multiple lines (apply this to other lines where you do this too).

When you pass through a boolean or some content which isn't easily identifyable, i.e. `, True)`, I would recommend using a keyword so it's clear what this is doing.

```self.vsphere_content.viewManager...(..., verify_ssl=True)``` for example